ha/k8s_setup/ha_setup.py

```diff
@@ -188,6 +188,10 @@ def process(self):
         Process config command.
         """
         try:
+            # Get log path from cluster.conf.
+            log_path = Conf.get(self._index, f'cortx{_DELIM}common{_DELIM}storage{_DELIM}log')
```

Comment:
Hello all,
is this the only place?
are these below places still in use?

https://github.com/Seagate/cortx-ha/blob/main/ha/const.py
below are constants:
RA_LOG_DIR="/var/log/seagate/cortx/ha"
PACEMAKER_LOG="/var/log/pacemaker.log"
AUTH_DIR="/var/lib/pcsd"
PCSD_LOG="/var/log/pcsd/pcsd.log"
HA_CMDS_OUTPUT="{}/ha_cmds_output".format(RA_LOG_DIR)
COROSYNC_LOG="/var/log/cluster"
PCSD_DIR="/var/log/pcsd"
LOG_DIR="/var/log"
and in bellow files also we are using /var/log/seagate/...
https://github.com/Seagate/cortx-ha/tree/main/conf/script/v1/
https://github.com/seagate/cortx-ha/blob/main/jenkins/cicd/cortx-ha-cicd.sh

Reply:
I think if we do changes at these places, we need to test it. and that path needs to be tested differently. So, I think let's keep it as it is. No need to change anything here.

Reply:
@mukhtar-inamdar , For RA_LOG_DIR, I have already talked to Shriya and she will be creating ticket for that and the SUPPORT BUNDLE changes .
remaining constants are all for pacemaker deployment and not applicable to k8s 
cortx-ha-cicd.sh is only creating the folder so we are fine even if it remains there
conf/script/v1 is used for V1 release of packemaker based CORTX deployment and is not applicable to k8s

Reply:
thanks then marking is resolved